generator/metricbeat/{beat}/magefile.go

```diff
@@ -96,6 +121,20 @@ func CollectDocs() error {
 	return metricbeat.CollectDocs()
 }
 
+// GoTestUnit executes the Go unit tests.
+// Use TEST_COVERAGE=true to enable code coverage profiling.
+// Use RACE_DETECTOR=true to enable the race detector.
+func GoTestUnit(ctx context.Context) error {
+	return devtools.GoTest(ctx, devtools.DefaultGoTestUnitArgs())
+}
+
+// GoTestIntegration executes the Go integration tests.
+// Use TEST_COVERAGE=true to enable code coverage profiling.
+// Use RACE_DETECTOR=true to enable the race detector.
+func GoTestIntegration(ctx context.Context) error {
+	return devtools.GoTest(ctx, devtools.DefaultGoTestIntegrationArgs())
+}
```

Comment:
@fearful-symmetry so it seems that we had also a target for integration tests before, maybe I should include this target too in the changes for master/7.5. WDYT?

Reply:
I would think so, yah